release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -1182,6 +1190,23 @@ With this release, replacement control plane nodes are assigned to the correct i
 
 * Previously, when editing any pipeline in the {product-title} console, the correct data was not rendered in the *Pipeline builder* and *YAML view* configuration options. Because of this issue, you could not edit the pipeline in the *Pipeline builder*. With this update, data is parsed correctly and you can edit the pipeline using the builder. (link:https://issues.redhat.com/browse/OCPBUGS-5016[*OCPBUGS-5016*])
 
+[discrete]
+[id="ocp-4-13-cloud-etcd-operator-bug-fixes"]
+==== etcd Cluster Operator
+
+* Previously, the `ControlPlaneMachineSet` Operator attempted to recreate a control plane machine before the cluster bootstrapping completed. This resulted in the removal of the bootstrap node from the etcd cluster membership that caused etcd quorum loss and the cluster to go offline. With this update, `ControlPlaneMachineSet` Operator only recreates a control plane machine after the etcd Cluster Operator removes the bootstrap node. (link:https://issues.redhat.com/browse/OCPBUGS-10960[*OCPBUGS-10960*]
+
+[discrete]
+[id="ocp-4-13-hypershift-bug-fixes"]
+==== HyperShift
```

Comment:
Just one more nit I didn't notice before. Sorry @dfitzmau. I think this should be Hosted Control Planes @lahinson can you confirm?

Reply:
I did some digging and I think hosted control planes is what we should go with for it based on https://docs.openshift.com/container-platform/4.12/release_notes/ocp-4-12-release-notes.html#ocp-4-12-hcp

Reply:
Thanks, updated.